pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -78,6 +78,10 @@ func getVifFilePath(pid, name string) string {
 	return fmt.Sprintf(vifCacheFile, pid, name)
 }
 
+func GetNDPConnectionUnixSocketPath(pid string, ifaceName string) string {
```

Comment:
Since the method is use from both common and podinterface. I find it weird the method is defined in podinterface. I would expect the opposite.

Reply:
https://github.com/kubevirt/kubevirt/pull/4581/commits/6d1365809884cfad99c33acf133056610ab85fb7